packages/mw-log-summary/src/index.js

```diff
@@ -0,0 +1,40 @@
+import assign from 'lodash/assign';
+import pick from 'lodash/pick';
+
+function requestResponseLogBody(req, res) {
+  return { req, res: { body: res.body } };
+}
+
+function isRequest(req) {
+  return req.headers.type === 'request' || req.headers.type === 'reactive';
+}
+
+export default function mwLogSummary(req, next, options, log) {
```

Comment:
should have option (environment variable) to work without times measurements - as it might impact the performance

Reply:
not significantly compared to logging io performance